tests/testsuite/cargo_tree/deps.rs

```diff
@@ -1246,25 +1246,25 @@ foo v0.1.0 ([ROOT]/foo) [bar,default,dep_that_is_awesome,foo,other-dep]
         .run();
 
     p.cargo("tree --all-features")
-        .arg("--format={p}")
+        .arg("--format={p} satisfies {vr}")
         .with_stdout_data(str![[r#"
-foo v0.1.0 ([ROOT]/foo)
-├── dep v1.0.0
-├── dep_that_is_awesome v1.0.0
-└── other-dep v1.0.0
-    └── dep v1.0.0
+foo v0.1.0 ([ROOT]/foo) satisfies *
+├── dep v1.0.0 satisfies =1.0
+├── dep_that_is_awesome v1.0.0 satisfies >=1.0, <2
+└── other-dep v1.0.0 satisfies ^1.0
+    └── dep v1.0.0 satisfies =1.0
```

Comment:
both paths to `dep` involve the same version requirement.  It would help if they were different.